[2022-06-17 08:33:58,728] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dags_new_version.create_table manual__2022-06-17T08:30:04.092813+00:00 [queued]>
[2022-06-17 08:33:58,802] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dags_new_version.create_table manual__2022-06-17T08:30:04.092813+00:00 [queued]>
[2022-06-17 08:33:58,817] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-17 08:33:58,819] {taskinstance.py:1357} INFO - Starting attempt 3 of 4
[2022-06-17 08:33:58,820] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-17 08:33:58,960] {taskinstance.py:1377} INFO - Executing <Task(PostgresOperator): create_table> on 2022-06-17 08:30:04.092813+00:00
[2022-06-17 08:33:59,012] {standard_task_runner.py:52} INFO - Started process 2689 to run task
[2022-06-17 08:33:59,032] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dags_new_version', 'create_table', 'manual__2022-06-17T08:30:04.092813+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/user.py', '--cfg-path', '/tmp/tmp1bsmropo', '--error-file', '/tmp/tmp1u0rjld4']
[2022-06-17 08:33:59,069] {standard_task_runner.py:80} INFO - Job 114: Subtask create_table
[2022-06-17 08:34:00,126] {task_command.py:370} INFO - Running <TaskInstance: dags_new_version.create_table manual__2022-06-17T08:30:04.092813+00:00 [running]> on host adccabfb4a32
[2022-06-17 08:34:00,441] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dags_new_version
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-17T08:30:04.092813+00:00
AIRFLOW_CTX_TRY_NUMBER=3
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-17T08:30:04.092813+00:00
[2022-06-17 08:34:00,474] {base.py:68} INFO - Using connection ID 'postgre_sql' for task execution.
[2022-06-17 08:34:00,501] {dbapi.py:213} INFO - Running statement: 
        DROP TABLE IF EXISTS z1_user;
        CREATE TABLE z1_user(
            user_id SERIAL PRIMARY KEY,
            name VARCHAR(255),
            phonenumber VARCHAR(255),
            email VARCHAR(255),
            address VARCHAR(255),
            work VARCHAR(255),
            other TEXT
        )
        , parameters: None
[2022-06-17 08:34:00,901] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dags_new_version, task_id=create_table, execution_date=20220617T083004, start_date=20220617T083358, end_date=20220617T083400
[2022-06-17 08:34:01,035] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-17 08:34:01,186] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
